polkadot/node/subsystem-bench/src/lib/usage.rs

```diff
@@ -151,3 +170,10 @@ impl ResourceUsage {
 }
 
 type ResourceUsageCheck<'a> = (&'a str, f64, f64);
+
+#[derive(Debug, Serialize)]
+pub struct ChartItem {
```

Comment:
Is this a format our other tooling is expecting ?

Reply:
yes, added to the PR description